src/base/bittorrent/session.cpp

```diff
@@ -269,6 +288,8 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), m_isUTPEnabled, clampValue(0, 1))
```

Comment:
You should declare compatible enums (better enum classes) inside BitTorrent namespace and use it. E.g.:
```c++
enum class ChokingAlgorithm
{
    FixedSlotsChoker,
    RateBasedChoker
};
// and then...
switch (chokingAlgorithm()) {
case ChokingAlgorithm::FixedSlotsChoker:
    settingsPack.set_int(libt::settings_pack::choking_algorithm, libt::settings_pack::fixed_slots_choker);
    break;
case ChokingAlgorithm::RateBasedChoker:
    settingsPack.set_int(libt::settings_pack::choking_algorithm, libt::settings_pack::rate_based_choker);
    break;
}
```